docs/chain/collator.md

```diff
@@ -405,7 +405,40 @@ The corresponding methods can be found as an extrinisc under `parachainStaking >
 
 ## Monitoring
 
-TODO
+It would be ideal if monitoring is done from a different host however in cases of resource constraints it would be better to run these monitoring tools on the same machine where your collator is running than not having.The following Monitoring stack is recommended.
+You can either run grafana by your own or subscribe free cloud option here https://grafana.com/products/cloud/. 
+NB: if you want to use cloud option you need to expose prometheus to outside access by chaning the ports section to ```  - "9090:9090"``` and reverse proxying with webserver is recommended.
+
+- Prometheus
+- Grafana
+- Node exporter
+- Slack
+
+There are two types of metrics we are going to collect. **Node Metrics** and **Blockchain Metrics.**
+
+### Prerequisite
+```
+sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+sudo chmod +x /usr/local/bin/docker-compose
+sudo ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
+```
+Steps:
+
+1. Clone or download our monitoring template from this link  https://github.com/KILTprotocol/docs
+2. Change directory to the above cloned project's collator directory: ```cd docs/collator```
+3. Edit .env file to have your change grafana admin password
+4. Run the following command if you want to install only prometheus and node exporter
+   ```./start-collator.sh```
+5. If you want to install prometheus with grafana comment line 6 of ``` start-collator.sh``` and uncomment line 7 then run step 4
+6. if you want to install prometheus, grafana and your collator with docker-compose comment line 6  of ``` start-collator.sh``` and uncomment line 8 then run  step 4
```

Comment:
i think that all the references to lines etc are quite hard to follow. Also if we change the files this will all change too. We will most likely change the files without updating the guide and than it's even harder to follow.